Clients.ts

```diff
@@ -0,0 +1,58 @@
+import FineractAPI from "./FineractAPI";
+export interface ClientDatatables {
+    registeredTableName: string,
+    data: object
+}
+export interface ClientInfo {
+    officeId: number,
+    firstname: string,
+    lastname: string,
+    externalId?: string,
+    dateFormat?: string,
+    locale?: string,
+    active: boolean,
+    activationDate: string,
+    submittedOnDate?: string,
+    savingsProductId?: number,
+    datatables: Array<ClientDatatables>,
+    clientClassificationId?: number,
+    clientTypeId?: number,
+    dateOfBirth?: string,
+    genderId?: number,
+    legalFormId: number
+    mobileNo?: string,
+    accountNo?: string,
+    staffId?: number,
+    groupId?: number
+}
+export interface ClientCreateResponse {
+    officeId: number,
+    clientId: number,
+    resourceId: number,
+    savingsId: number,
+    groupId?: number
+}
+export default class Clients {
+    fineract_obj: FineractAPI
+    constructor(fineract_obj: FineractAPI) {
+        this.fineract_obj = fineract_obj;
+    }
+    // async list_clients(config?: LoanListConfig): Promise<[LoanInterface]> {
```

Comment:
Why this is commented out ?

we should also add `list_clients` function in this class.